### PR TITLE
RIFEX-139, UI Refactor on Config Page

### PR DESCRIPTION
**Pull request by fmelo**
This PR contains all the styles and markup for the refactor on the Configuration Page

### CODE REVIEW

File: ui/app/rif/pages/configuration/index.js [hunk 1]
```diff
@@ -131,51 +141,64 @@ class RifConfiguration extends Component {
 
   render () {
     if (this.state.loading) {
-      return (<div>Loaing Configuration...</div>);
+      return (<div className="">Loaing Configuration...</div>);
```
Comment (rorolopetegui) [inline ui/app/rif/pages/configuration/index.js]: Are we putting some classNames here? Or just leave it empty? Delete it in that case.

**Review by rorolopetegui — CHANGES_REQUESTED** [review]
Just a minor change, lgtm

File: ui/app/rif/pages/configuration/index.js (hunk 1 above)
Comment (fmelo) [reply ui/app/rif/pages/configuration/index.js]: Done

**Review by JONAF2103 — APPROVED** [review]
Good job, LGTM

**Pull request merged**